inf-ruby.el

```diff
@@ -165,7 +166,7 @@ graphical char in all other prompts.")
 (defvar inf-ruby-mode-map
   (let ((map (copy-keymap comint-mode-map)))
     (define-key map (kbd "C-c C-l") 'ruby-load-file)
-    (define-key map (kbd "C-x C-e") 'ruby-send-last-sexp)
+    (define-key map (kbd "C-x C-e") 'ruby-send-last-stmt)
```

Comment:
Please rebase on master, for better attribution and separation of changes.

I've pushed the new command and the changes in bindings.

Reply:
Nice, I've rebased thanks.